### PR TITLE
Change Personal Page to My Reviews

### PR DESCRIPTION
**Pull request by StephenZeiler**
This PR focuses on changing the Personal Page to My Reviews. 
Changes made: 
Renamed variables in front end with "Personal" to "My Reviews"
Changed Icon on My Reviews Page.

### CODE REVIEW

File: react-locate-contractor/src/config.tsx [hunk 1]
```diff
@@ -3,14 +3,14 @@ export default {
         issuer: 'https://dev-81535986.okta.com/oauth2/default',
         clientId: '0oa5se2c6gLXwYT2S5d7',
         scopes: ['openid', 'profile', 'email'],
-        redirectUri: `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`
-        // redirectUri: `http://localhost:3000/login/callback`
+        // redirectUri: `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`
+        redirectUri: `http://localhost:3000/login/callback`
```
Comment (zacoope1) [inline react-locate-contractor/src/config.tsx]: Is this going to break your cloud api instance?

File: react-locate-contractor/src/config.tsx [hunk 2]
```diff
@@ -3,14 +3,14 @@ export default {
...
     },
     widget: {
         issuer: 'https://dev-81535986.okta.com/oauth2/default',
         clientId: '0oa5se2c6gLXwYT2S5d7',
-        redirectUri: `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`,
-        // redirectUri: `http://localhost:3000/login/callback`,
+        // redirectUri: `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`,
+        redirectUri: `http://localhost:3000/login/callback`,
```
Comment (zacoope1) [inline react-locate-contractor/src/config.tsx]: Is this going to break your cloud api instance?

**Review by zacoope1 — CHANGES_REQUESTED** [review]

File: react-locate-contractor/src/config.tsx (hunk 1 above)
Comment (StephenZeiler) [reply react-locate-contractor/src/config.tsx]: Yes it would, I updated the config file.

File: react-locate-contractor/src/Pages/Login.tsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ const Login = ({ config }: any) => {
     return authState.isAuthenticated ?
         <Navigate to={{ pathname: '/' }} /> :
         <SignInWidget config={config} onSuccess={onSuccess} onError={onError} />;
+
```
Comment (Krescenskok) [inline react-locate-contractor/src/Pages/Login.tsx]: I would remove this line since this file only differs by whitespace.

**Review by zacoope1 — APPROVED** [review]

**Pull request merged**